README.md

```diff
@@ -273,6 +273,14 @@ request.get('http://some.server.com/', {
     'bearer': 'bearerToken'
   }
 });
+// or with SPNEGO/Kerberos
+request.get('http://some.server.com/').auth(null, null, false, null, true);
```

Comment:
I agree, the current auth() function signature isn't the most extensible especially if we want to add additional methods.